server/simple/public/index.html

```diff
@@ -26,14 +26,29 @@
   img {
     max-width: 70vw;
   }
+
+  code {
+    background: #eee;
+    padding: 3px;
+  }
 </style>
 
 <body>
 <h1>Welcome to Dashland!</h1>
 <img src="/dashland.jpeg" alt="Dashland">
-<p>This file is being served by <a href="https://pub.dev/packages/shelf_static">package:shelf_static</a> from the <code>/public</code> directory.</p>
+<p>This file is being served by <a
+    href="https://pub.dev/packages/shelf_static">package:shelf_static</a> from the
+  <code>/public</code> directory.</p>
 <!-- TODO: Link to new pkg:shelf docs once https://github.com/dart-lang/site-www/issues/3058 is fixed -->
-<p>There are code-based handlers at <a href="/helloworld">/helloworld</a> and <a href="/time">/time</a></p>
+<p>There are several code-based handlers defined using <a
```

Comment:
Can we clarify this? _There are_ is a red flag for content that might be clearer... and it could be. :) Do you mean this:

> This app uses package:shelf_router to define several code-based handlers